javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/AbstractSymbolResolutionTest.java

```diff
@@ -21,13 +21,24 @@
 
 package com.github.javaparser.symbolsolver;
 
-import com.github.javaparser.utils.CodeGenerationUtils;
-
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
+import org.junit.jupiter.api.AfterEach;
+
+import com.github.javaparser.ParserConfiguration;
+import com.github.javaparser.StaticJavaParser;
+import com.github.javaparser.utils.CodeGenerationUtils;
+
 public abstract class AbstractSymbolResolutionTest {
+    
+    @AfterEach
+    public void reset() {
```

Comment:
I decided to create a test parser somewhere, I was using that lately. I'm not sure what it is called. Better not use StaticJavaParser for the reason you mention.